source/addonHandler/__init__.py

```diff
@@ -886,234 +387,23 @@ def initTranslation():
-		displayName = string()
-		contracted = boolean(default=false)
-		input = boolean(default=true)
-		output = boolean(default=true)
-
-# Symbol Pronunciation
-[symbolDictionaries]
-	# The key is the symbol dictionary file name (not the full path)
-	[[__many__]]
-		displayName = string()
-		mandatory = boolean(default=false)
-
-# NOTE: apiVersion:
-# EG: 2019.1.0 or 0.0.0
-# Must have 3 integers separated by dots.
-# The first integer must be a Year (4 characters)
-# "0.0.0" is also valid.
-# The final integer can be left out, and in that case will default to 0. E.g. 2019.1
-
-""",
-		),
-	)
-
-	def __init__(self, input: IO[bytes], translatedInput: IO[bytes] | None = None):
-		"""Constructs an :class:`AddonManifest` instance from manifest string data
-
-		:param input: data to read the manifest information
-		:param translatedInput: Optional translated manifest input, defaults to ``None``
-		"""
-		super().__init__(input, configspec=self.configspec, encoding="utf-8", default_encoding="utf-8")
-		self._errors = None
-		val = Validator({"apiVersion": validate_apiVersionString})
-		result = self.validate(val, copy=True, preserve_errors=True)
-		if result != True:  # noqa: E712
-			self._errors = result
-		elif True != self._validateApiVersionRange():  # noqa: E712
-			self._errors = "Constraint not met: minimumNVDAVersion ({}) <= lastTestedNVDAVersion ({})".format(
-				self.get("minimumNVDAVersion"),
-				self.get("lastTestedNVDAVersion"),
-			)
-		self._translatedConfig = None
-		if translatedInput is not None:
-			self._translatedConfig = ConfigObj(translatedInput, encoding="utf-8", default_encoding="utf-8")
-			for k in ("summary", "description"):
-				val = self._translatedConfig.get(k)
-				if val:
-					self[k] = val
-			for fileName, tableConfig in self._translatedConfig.get("brailleTables", {}).items():
-				value = tableConfig.get("displayName")
-				if value:
-					self["brailleTables"][fileName]["displayName"] = value
-			for fileName, dictConfig in self._translatedConfig.get("symbolDictionaries", {}).items():
-				value = dictConfig.get("displayName")
-				if value:
-					self["symbolDictionaries"][fileName]["displayName"] = value
-
-	@property
-	def errors(self):
-		return self._errors
-
-	def _validateApiVersionRange(self):
-		lastTested = self.get("lastTestedNVDAVersion")
-		minRequiredVersion = self.get("minimumNVDAVersion")
-		return minRequiredVersion <= lastTested
-
-
-def validate_apiVersionString(value: str) -> Tuple[int, int, int]:
-	"""
-	@raises: configobj.validate.ValidateError on validation error
-	"""
-	from configobj.validate import ValidateError
-
-	if not value or value == "None":
-		return (0, 0, 0)
-	if not isinstance(value, string_types):
-		raise ValidateError('Expected an apiVersion in the form of a string. EG "2019.1.0"')
-	try:
-		return addonAPIVersion.getAPIVersionTupleFromString(value)
-	except ValueError as e:
-		raise ValidateError('"{}" is not a valid API Version string: {}'.format(value, e))
+__all__ = [
```

Comment:
>Manually verified that addon functionality still works correctly

Can you confirm how you tested this? did you check the diff of `dir(addonHandler)` from the NVDA python console?